src/Conversion/ONNXToTorch/NN/PaddingOp.cpp

```diff
@@ -142,31 +136,29 @@ class ONNXConstantPadNdOpToTorchLowering : public ConversionPattern {
     auto dataType = Torch::ValueTensorType::get(
         context, dataTensorType.getShape(), dataTensorType.getElementType());
 
-    auto dataTorchTensor =
+    Value dataTorchTensor =
```

Comment:
This can probably be replaced with the type converter. I would use the operand adaptor to avoid these casts.